chroma_core/services/job_scheduler/job_scheduler.py

```diff
@@ -867,9 +868,11 @@ def notify(self, content_type, object_id, time_serialized, update_attrs, from_st
 
     def run_jobs(self, job_dicts, message):
         with self._lock:
-            result = self.CommandPlan.command_run_jobs(job_dicts, message)
-            self.progress.advance()
-            return result
+            with transaction.atomic():
```

Comment:
Why the atomic here?

Reply:
I noticed it being used in other places, such as "set_host_profile" so I thought I should probably add it here as well. Will this cause an issue?

Reply:
It really should only be added if there is a clear reason to do so.

Reply:
ok, i'll remove it.